go/auth/credential_authority.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 const (
-	userTimeout  = 5 * time.Minute
-	cacheTimeout = 8 * time.Minute
+	userTimeout  = 3 * time.Hour
+	cacheTimeout = 8 * time.Hour
```

Comment:
@maxtaco So it look like we have to increase `cacheTimeout` too. Otherwise `user` entries get evicted too soon and fail test (and I imagine it makes increasing `userTimeout` a moo-point too). Let me know if you think there's a problem with increasing both. Thanks!